exonum-java-binding/core/rust/integration_tests/tests/node_context.rs

```diff
@@ -78,13 +78,10 @@ fn submit_transaction_to_missing_service() {
 }
 
 fn create_transaction(instance_id: u32) -> AnyTx {
-    AnyTx {
-        call_info: CallInfo {
-            instance_id,
-            method_id: TEST_TRANSACTION_ID,
-        },
-        arguments: TEST_TRANSACTION_PAYLOAD.to_vec(),
-    }
+    AnyTx::new(
+        CallInfo::new(instance_id, TEST_TRANSACTION_ID),
```

Comment:
Split to two statements might be more readable... 🤔 I'll add a note to make such refactoring.